go/client/logclient.go

```diff
@@ -133,7 +133,7 @@ func (c *LogClient) fetchAndParse(uri string, res interface{}) error {
 	defer ioutil.ReadAll(resp.Body)
 
 	if resp.StatusCode != 200 {
-		return errors.New("status code not 200")
+		return errors.New("status code not 200: " + strconv.Itoa(resp.StatusCode))
```

Comment:
`fmt.Sprintf()` was exactly the same, but there's a `fmt.Errorf()` that seems to fit perfectly. I made the message similar to another one at line 237, while I was there, for consistency.